lib/api/redfish-1.0/odata.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017, EMC, Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var getOdata = controller(function() {
+    var data = {
+        "@odata.context": "/redfish/v1/$metadata",
```

Comment:
following redfish spec for odata http://redfish.dmtf.org/schemas/DSP0266_1.1.html#odata-service-document